prowler/providers/github/services/repository/repository_public_has_securitymd_file/repository_public_has_securitymd_file.metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+  "Provider": "github",
+  "CheckID": "repository_public_has_securitymd_file",
+  "CheckTitle": "Check if public repositories have a SECURITY.md file",
+  "CheckType": [
+    "CIS GitHub Benchmark v1.0.0"
```

Comment:
Remove please. 
We should plan a different way to write the compliance that the check belongs to.